publiccode.yml

```diff
@@ -1,6 +1,6 @@
 publiccodeYmlVersion: '0.2'
 name: Signalen frontend
-url: 'https://github.com/signalen/frontend'
+url: 'https://github.com/Amsterdam/signals-frontend'
```

Comment:
What is the practical implication of this change? 

Is it possible to instead use the `isBasedOn` key to refer back to this (Amsterdam/signals-frontend) repository?

Reply:
@MauritsR the [Developers Italia crawler](https://github.com/italia/developers-italia-backend) has an additional check on the `url` key:

```
url: declared url (https://github.com/signalen/frontend) and actual publiccode.yml source URL
(https://raw.githubusercontent.com/Amsterdam/signals-frontend/develop/) are not in the same repo:
'//github.com/signalen/frontend' vs '//github.com/Amsterdam/signals-frontend
```

and the repo added to the crawling list was this one: https://github.com/italia/developers-italia-backend/pull/238

I'm not sure about the relationship between the two repos. Will they always be mirrored or will they diverge? Which one will be developed in the future?

My guess is that we should add the main one ([signalen/frontend](https://github.com/signalen/frontend)?) to the crawling list

Reply:
@bfabio The repositories are mirrored. As of now, the Amsterdam/signals-frontend repository is leading - but there are plans to change this in the future.

However, since this repository is leading right now it is correct to change `url` to `https://github.com/Amsterdam/signals-frontend`.